core/commands/dag/import.go

```diff
@@ -159,17 +159,12 @@ func importWorker(req *cmds.Request, re cmds.ResponseEmitter, api iface.CoreAPI,
 		err := func() error {
 			defer file.Close()
 
-			car, err := gocar.NewCarReader(file)
+			car, err := gocarv2.NewBlockReader(file)
 			if err != nil {
 				return err
 			}
 
-			// Be explicit here, until the spec is finished
-			if car.Header.Version != 1 {
-				return errors.New("only car files version 1 supported at present")
-			}
-
-			for _, c := range car.Header.Roots {
+			for _, c := range car.Roots {
```

Comment:
This seems reasonable to me. @ribasushi @willscott any gotchas here I need to be aware of?

Reply:
This should the expected thing and read through the roots & blocks of either a v1 or v2 car